.emacs.d/lisp/package/packup.el

```diff
@@ -47,7 +229,8 @@ If UPDATE is non-nil, out-of-date packages will be updated."
                    (terpri)
                    (princ (symbol-name (car package)))
                    (princ (if (cdr package) " (update)" " (install)"))))
-               (if (y-or-n-p "Auto install/update these package?")
+               (if (or auto-update
+                       (y-or-n-p "Auto install/update these package?"))
```

Comment:
`Auto install/update these package(s)?`

Reply:
This is part of existing code. Open an issue.